### PR TITLE
a2ping: add page

### LINKED ISSUE #10884: page request: a2ping
### Command description

a2ping is a script that can convert images into EPS or PDF files. See https://www.ctan.org/pkg/a2ping

### Command details

_No response_

### OS

- [ ] Android
- [X] Linux
- [ ] macOS
- [ ] sunOS
- [X] Windows

### PR DESCRIPTION
**Pull request by npv12**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 2.84p

fixes #10884

### CODE REVIEW

File: pages/linux/a2ping.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+> Convert images into EPS or PDF files.
+> More information: <https://www.ctan.org/pkg/a2ping>.
+
+- Convert an image to PDF. The output file name is optional:
+
+`a2ping {{filename}} {{outputFilename.pdf}}`
+
+- Compress the document. Possible options: none, zip, best, flate:
+
+`a2ping --[no]compress {{option}} {{filename}}`
```
Comment (gutjuri) [inline pages/linux/a2ping.md:12]:
```suggestion
- Compress the document using the specified method:

`a2ping --[no]compress {{none|zip|best|flate}} {{filename}}`
```

**Comment by kbdharun** [comment]
Marked this PR as draft, as the tool seems to be cross-platform so I will move it to `common` and remove the platform-specific pages and also apply @gutjuri's suggestion.

**Comment by npv12** [comment]
@kbdharun I have updated the MR correspondingly

**Review by gutjuri — APPROVED** [review]

**Comment by kbdharun** [comment]
> @kbdharun I have updated the MR correspondingly

Great, I will review this PR later today.

**Comment by kbdharun** [comment]
Changes I made:

- Updated the link from the author's site to the man page (as the author's site documentation doesn't explain the usage in detail).
- Updated the path placeholders.
- Updated descriptions and used "Note:" for secondary descriptions.
- Change the help description to our generic "Display Help:" description.
- Fixed options [see ref]

Ref.

```zsh
➜  Downloads a2ping --(no)below
zsh: no matches found: --(no)below
➜  Downloads a2ping --nobelow  
a2ping.pl 2.84p, 2019-11-17 -- Written by <pts@fazekas.hu> from April 2003.
This is free software, GNU GPL >=2.0. There is NO WARRANTY.
(epstopdf 2.7 Copyright 1998-2001 by Sebastian Rahtz et al.)

Usage:  a2ping.pl [options] <inputfile> [[<outformat>:] <outputfile>]
Run with --doc to read documentation as a UNIX man(1) page.
Options: --help print this help message
--(no)compress  use compression                   (def: best)
--(no)hires     scan HiResBoundingBox             (def: yes)
--(no)exact     scan ExactBoundingBox             (def: no)
--(no)verbose   verbose debug informations        (def: no)
--(no)below     allow below+left_from baseline    (def: no)
--(no)tmpunlink unlink temporary files            (def: yes)
--(no)antialias render shades at outlines (def: scale3no) (=scale3yes =no =yes)
--(no)lossy     allow lossy image filters (EPS->PDF) (def: yes)
--(no)keepoldmediabox keep only old, [0 0]-based MediaBox in PDF (def: no)
--gs-cmd=       path to Ghostscript               (def: gs or gswin32c)
--gs-ccmd=      path to Ghostscript, 4 bbox calc  (def: gs or gswin32c)
--gsextra=      extra arg to gs
--extra=        extra arg to external prg (i.e pdftops)
--bboxfrom=     adsc|compute-gs|pagesize          (def: guess)
--papersize=    unchanged|force-unknown|600bpx5cm (def: default) (bp)
--threshold=    min color for 1 in 8->1 bit conv  (def: 128)
Possible input formats: PS EPS PDF JPEG GIF TIFF PNG PNM PCX BMP LBM XPM TGA
Possible output formats: BMP EPS GIF JPEG PBM PCL5 PDF PDF1 PGM PNG PPM PS TIFF XPM XWD markedEPS markedPS
Examples for producing 'test.pdf':
  * a2ping.pl test.eps
  * produce postscript | a2ping.pl -v - test.pdf
Example: look for HiResBoundingBox and produce corrected PostScript:
  * a2ping.pl -d --nogs -hires test.ps>testcorr.ps
!!! Error: Input filename missing
```

**Comment by kbdharun** [comment]
@npv12 Feel free to recheck the page and let me know if you are fine with the changes.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**